Core/Src/porting/odroid_overlay.c

```diff
@@ -858,7 +856,7 @@ int odroid_overlay_confirm(const char *text, bool yes_selected)
         {0, curr_lang->s_No, "", 1, NULL},
         ODROID_DIALOG_CHOICE_LAST,
     };
-    return odroid_overlay_dialog(curr_lang->s_PlsChose, choices, yes_selected ? 2 : 3);
+    return odroid_overlay_dialog(curr_lang->s_PlsChose, choices, yes_selected ? 2 : 3, NULL); //TODO add repaint callback
```

Comment:
still todo ?

Reply:
I suggest that we keep the TODO as is and I will revisit this at a later date not too far in the future